Makefile

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 The Fission Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.DEFAULT_GOAL := multiarch-all
+
+# Platforms to build in multi-architecture images.
+PLATFORMS ?= linux/amd64,linux/arm64,linux/arm/v7
+
+# Repository prefix and tag to push multi-architecture images to.
+REPO ?= fission
+TAG ?= dev
+
+verify-builder: 
+	@./hack/buildx.sh 
+
+multiarch-all: multiarch-binary multiarch-go multiarch-python multiarch-nodejs multiarch-perl multiarch-php multiarch-ruby
+
+multiarch-binary: verify-builder multiarch-binary-env multiarch-binary-builder
+
+multiarch-go: verify-builder multiarch-go-env multiarch-go-builder
+
+multiarch-python: verify-builder multiarch-python-env multiarch-python-builder
+
+multiarch-nodejs: verify-builder multiarch-nodejs-env multiarch-nodejs-builder
+
+multiarch-perl: verify-builder multiarch-perl-env 
+
+multiarch-php: verify-builder multiarch-php-env  multiarch-php-builder
+
+multiarch-ruby: verify-builder multiarch-ruby-env multiarch-ruby-builder
+
+multiarch-binary-env: 
+	docker buildx build --platform=$(PLATFORMS) -t $(REPO)/binary-env:$(TAG) --push -f binary/Dockerfile binary/
```

Comment:
Can we add below TAG definition above ?
```
PUSH ?= --push
```
We can use it like 
```
docker buildx build --platform=$(PLATFORMS) -t $(REPO)/binary-env:$(TAG) $(PUSH) -f binary/Dockerfile binary/
```

Benefit of doing this would be , it would make push optional.
```
PUSH= make multiarch-ruby

PUSH="--load" make multiarch-ruby # for local build

PUSG="--push" make  multiarch-ruby
```

This would apply to all docker buildx commands.

Reply:
`--load` only works when we have specified the host arch. 
`PUSH=--load PLATFORMS=linux/amd64 make multiarch-binary-env`